src/gsi/constants.f90

```diff
@@ -90,7 +90,7 @@ module constants
 
 ! Declare derived constants
   integer(i_kind):: huge_i_kind
-  integer(i_kind), parameter :: max_varname_length=64
+  integer(i_kind), parameter :: max_varname_length=20
```

Comment:
@jderber-NOAA In my regional run, I just the max_varname-length=20 is not enough to contain names like" fv3SAR01_ens_mem001-fv3_dynvars" . 
^
character(len=max_varname_length) :: filenamein2
V
 Would you consider changing this to a bigger value like original one? Or I should use a different size (another parameter) for my use? 
````
Thanks.

Reply:
Yes.  It is in the list of changes I am currently working on.

Reply:
John, Thanks a lot!